client/src/partials/patients/documents/modals/documents.modal.html

```diff
@@ -0,0 +1,70 @@
+<div class="modal-header">
+  <span class="glyphicon glyphicon-paperclip"></span>
+  {{ 'PATIENT_DOCUMENT.ADD' | translate }} {{ 'PATIENT_DOCUMENT.DOCUMENTS' | translate }}
+  <span class="label label-primary"> {{ $ctrl.patient.abbr + $ctrl.patient.reference }}</span>
```

Comment:
In the `Patients.list()` route, we have the following in the SQL query: `CONCAT(pr.abbr, p.reference) AS reference`.  [Here is the line of code](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/server/controllers/medical/patients/index.js#L233).  However, I noticed that this `CONCAT` is missing from the `Patients.detail()` route.

Could you add it to the `Patient.detail()` route, and then you will be able to simply write `{{ $ctrl.patient.reference }}` in your template?  Be careful to check that this suggestion doesn't break other modules!